cms/__init__.py

```diff
@@ -137,6 +139,20 @@
     LANG_PASCAL: ".o",
 }
 
+COMPILED = 1
+HALF_COMPILED = 3
+INTERPRETED = 5
```

Comment:
OK, I'll do some timing over the week for it and find some semi-decent values, added a card on Trello for making it better